taichi/analysis/data_source_analysis.cpp

```diff
@@ -37,6 +37,10 @@ std::vector<Stmt *> get_load_pointers(Stmt *load_stmt) {
     return external_func->arg_stmts;
   } else if (auto ref = load_stmt->cast<ReferenceStmt>()) {
     return {ref->var};
+  } else if (auto matrix_init = load_stmt->cast<MatrixInitStmt>()) {
+    return matrix_init->values;
```

Comment:
Why is `matrix_init->values` a pointer?

Reply:
Same as below (can be removed according to current cfg pass support for new matrix)